Formula/zpaq.rb

```diff
@@ -1,27 +1,49 @@
 class Zpaq < Formula
   desc "Incremental, journaling command-line archiver"
   homepage "http://mattmahoney.net/dc/zpaq.html"
-  url "http://mattmahoney.net/dc/zpaq705.zip"
-  sha256 "d8abe3e3620d4c6f3ddc1da149acffa4c24296fd9c74c9d7b62319e308b63334"
-  version "7.05"
-
+  url "http://mattmahoney.net/dc/zpaq713.zip"
+  version "7.13"
+  sha256 "9120cf4fb1afdecea3ac4f690d7b0577f7cb004ca6b152856edd8ac444f0d919"
   head "https://github.com/zpaq/zpaq.git"
 
   bottle do
-    cellar :any_skip_relocation
-    sha256 "f0998826effe4dae407d041c4143a39b3645dc46a82456b4e86ac88f106babd3" => :el_capitan
-    sha256 "bdd4716595b8eed1c52d2a2ef43abb095a996d05a0489f81f0b22ae43a26d51d" => :yosemite
-    sha256 "a71ad4808da8c69844f72322ff7633b895d034028129b30c3d25409d91bd4213" => :mavericks
-    sha256 "a2d2215b51cc18370f5ebe700160d1d7143ce5d680b2365a5adb4bb1622d06fc" => :mountain_lion
+    cellar :any
+    sha256 "c75fda6498ab5c8535654bd1f8057b6fec6717bf62daab743e5fb0142bead204" => :el_capitan
   end
 
   def install
-    system "make"
-    include.install "libzpaq.h"
-    bin.install "zpaq"
+    # Makefile introduced for zpaq >= 7.06 was never adapted for OS X
+    # Reported 9th May 2016: https://github.com/zpaq/zpaq/pull/13
```

Comment:
It is. It's just adding the same file generated after these replacements. Including the two "unnecessary" ones that are commented out. But it would need to go in patches repo since apparently the `head` url in this formula only points to a mirror, and they already closed the PR: https://github.com/zpaq/zpaq/pull/13#issuecomment-217909974

I'm wondering if the head url should be removed if it's just a mirror, but upstream does link to it on his site http://mattmahoney.net/dc/zpaq.html where it's described, as follows: "Github Repository maintained by Evan Nemerson and Yonggang Luo. Includes history and convert script."

I suppose I'll need to email upstream directly.